tests/conftest.py

```diff
@@ -14,19 +14,59 @@
 
 import pytest
 
+from model import storage
 from model.app import app as app_
 from model.app import init_app
 
 
 @pytest.fixture(scope="session")
 def app():
-    """Provide an initialized Flask for use in certain test cases."""
+    """Provide the initialized Flask app"""
     init_app(app_, None)
     return app_
 
 
 @pytest.fixture(scope="session")
 def client(app):
-    """Provide a Flask test client to be used by almost all test cases."""
+    """Provide a Flask test client"""
     with app.test_client() as client:
         yield client
+
+
+@pytest.fixture(scope="function")
+def e_coli_core():
+    """
+    Provide a modifiable copy of the e_coli_core cobrapy model.  This model is fairly small and should be preferred in
+    test cases where possible.
+    """
+    model = storage.get('e_coli_core').model
+
+    # The context processor will not reset the `notes` field on exit, so keep a copy and reset it manually
+    original_notes = model.notes
+    with model:
+        yield model
+    model.notes = original_notes
```

Comment:
Please just look into whether this resets the notes (I would be really surprised) or why the tests pass anyway right now if notes are accumulating here.